Tests/FunctionalTests/MiscellaneousTests.swift

```diff
@@ -440,23 +437,27 @@ class MiscellaneousTestCase: XCTestCase {
                 guard let output = String(bytes: bytes, encoding: .utf8) else {
                     return
                 }
+                print(output, terminator: "")
                 self.lock.withLock {
                     switch self.state {
                     case .idle:
-                        self.state = .buffering(output)
+                        self.state = processOutput(output)
```

Comment:
Just curious if this could also have been done with a `fallthrough` here?

Reply:
The separate function seems cleaner, arguably, mostly curious if I understand the logic here correctly.

Reply:
cannot use `fallthrough ` afaik since we need to bind the variable